lib/game_teams.rb

```diff
@@ -0,0 +1,22 @@
+class GameTeams
```

Comment:
This is what Kathy and I worked on yesterday.  It is designed to standardize the headers of csv files.